tests/spreadsheet/spreadsheet.html

```diff
@@ -338,7 +338,7 @@
                                 <div class="k-vertical-align-center">Amount</div>
                             </div>
                             <div class="k-spreadsheet-cell" style="background: rgb(167, 214, 255); outline: rgb(150, 193, 230) solid 1px; color: rgb(0, 62, 117); font-family: Arial; text-align: center; font-size: 14px; white-space: pre; overflow-wrap: normal; left: 816px; top: 71px; width: 63px; height: 24px;"></div>
-                            <div class="k-spreadsheet-cell k-spreadsheet-active-cell k-left k-single" style="background: rgb(255, 255, 255); outline: rgb(230, 230, 230) solid 1px; color: rgb(0, 62, 117); font-family: Arial; text-align: center; font-size: 14px; white-space: pre; overflow-wrap: normal; left: 1px; top: 96px; width: 99px; height: 29px;">
+                            <div class="k-spreadsheet-cell k-left k-single k-spreadsheet-active-cell" style="background: rgb(255, 255, 255); outline: rgb(230, 230, 230) solid 1px; color: rgb(0, 62, 117); font-family: Arial; text-align: center; font-size: 14px; white-space: pre; overflow-wrap: normal; left: 1px; top: 96px; width: 99px; height: 30px;">
```

Comment:
the height difference probably causes the misalignment of the selection in the related test